pe/reporter.py

```diff
@@ -81,7 +81,6 @@ def prepare_context(self) -> None:
 
         support_email: str = os.getenv('SUPPORT_EMAIL', 'its.tl.staff@umich.edu')
         self.context = {'report': report_dict, 'exams': exam_dicts, 'support_email': support_email}
-        LOGGER.debug(self.context)
```

Comment:
I'm removing this since it usually floods the console with data, which was getting in the way of me looking at earlier log messages.